src/app/forms/forms.module.js

```diff
@@ -1,5 +1,137 @@
 (function() {
   'use strict';
 
-  angular.module('uvpBuilderWeb.forms', []);
+  angular.module('uvpBuilderWeb.forms', ['flow', 'ui.router'])
+    .config(['flowFactoryProvider', function (flowFactoryProvider) {
+      flowFactoryProvider.defaults = {
+        target: 'app/forms/upload.php',
+        permanentErrors: [404, 500, 501],
+        maxChunkRetries: 1,
+        chunkRetryInterval: 5000,
+        simultaneousUploads: 4,
+        singleFile: true
+      };
+    }])
+    .config(function ($stateProvider) {
```

Comment:
Same thing about the `$inject`; and you could move the routes to a diffent file, `forms.route.js`